src/pal/inc/pal.h

```diff
@@ -6870,6 +6870,7 @@ class NativeExceptionHolderFactory
     };                                                                          \
     try                                                                         \
     {                                                                           \
+        HardwareExceptionHolder                                                 \
```

Comment:
I can see that you have removed the CatchHardwareExceptionHolder base class from the NativeExceptionHolderBase, but you have added it to the macros at all places where we have the native holder. So doesn't the change result the same behavior as before?